backend/internal/models/penalty.go

```diff
@@ -2,19 +2,19 @@ package models
 
 type PenaltyType struct {
 	DbModel
-	Name     string `json:"name"`
 	Duration uint   `json:"duration"`
+	Name     string `json:"name"`
 	Severity string `json:"severity"`
 }
 
 type Penalty struct {
 	DbModel
-	PlayerID      uint        `json:"player_id"`
-	TeamID        uint        `json:"team_id"`
-	GameID        uint        `json:"game_id"`
-	Period        uint        `json:"period"` // TODO: how to represent Shootouts?
-	Duration      uint        `json:"duration"`
 	CreatedBy     uint        `json:"created_by"`
+	Duration      uint        `json:"duration"`
+	GameID        uint        `json:"game_id"`
 	PenaltyType   PenaltyType `json:"penalty_type"`
 	PenaltyTypeID uint        `json:"penalty_type_id"`
+	Period        uint        `json:"period"` // TODO: how to represent Shootouts?
+	PlayerID      uint        `json:"player_id"`
+	TeamID        uint        `json:"team_id"`
```

Comment:
I've added `gorm:"not_null"` to all of these properties besides Duration in my branch, let me know the timeline on when you want to merge this and who should go first.

Reply:
Or if you want to add those changes

Reply:
Also we don't need the Shootouts comment anymore, since we won't have penalties in Shootouts

Reply:
I'll merge my changes first